main.go

```diff
@@ -34,9 +36,67 @@ func main() {
 	if version == "" {
 		version = "2-dev"
 	}
+	// Orange to Yellow gradient colors (reversed order)
+	gradientColors := []string{
+		"\033[38;5;202m",
+		"\033[38;5;208m", // Orange
+		"\033[38;5;202m", // Red
+		"\033[38;5;214m", // Yellow-Orange
+	}
 	utils.Version = version
 	if binaryToDocker := os.Getenv("BINARY_TO_DOCKER"); binaryToDocker != "true" {
-		fmt.Println(logo, " ")
+		// ANSI escape code to reset color
+		const reset = "\033[0m"
+
+		// Print each line of the logo with a different color from the gradient
+		lines := strings.Split(logo, "\n")
+		for i, line := range lines {
+			for j, char := range line {
+
+				var color string
+				if i == 7 {
+					if j > 42 {
+						color = gradientColors[0]
+					} else {
+						color = gradientColors[3]
```

Comment:
would suggest some more in-detail

<img width="617" alt="image" src="https://github.com/keploy/keploy/assets/60891544/10c928cb-5a80-4923-b09d-1371328d7532">
Kind of sample cli images can be built via the ANSCII codes. So something that resembles more with Keploy's logo would be possible too,